### DISCUSSION TITLE
이전 글/다음 글 X 한 번에 안됨

### DISCUSSION THREAD

**Discussion by MJKim99** [pronist/hELLO#153] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 10
- 브라우저: Chrome 107.0.5304.122
- 스킨 버전  3.6.5

**설명**
다크모드 때 이전 글/다음 글 안내가 노출되었을 때, X 표시를 눌렀으나 한 번에 사라지지 않습니다.
2~5번 이상 연속 클릭해줘야 사라집니다.

제 컴퓨터 문제인가 싶어 캐시 비우기 및 강력 새로고침도 해보았으나 똑같았고,
크롬말고 엣지를 사용해도 같은 문제가 있었으며, 다른 컴퓨터에서도 시도해보았으나 같은 문제가 있었습니다.

기분 탓인지 모르겠으나 라이트모드 때보다 다크모드 때가 더 안되는 것 같은 느낌이었습니다.

**스크린샷 및 링크**
![Animation](https://user-images.githubusercontent.com/81893393/204098508-f061c1d1-d3c7-41c6-8f34-aafe4dadf912.gif)
https://mimah.tistory.com/entry/%EC%9C%88%EB%8F%84%EC%9A%B0-%EA%B4%80%EB%A6%AC%EC%9E%90-%EB%AA%A8%EB%93%9C-cmd%EC%97%90%EC%84%9C-%EC%82%AC%EC%9A%A9-%EC%A4%91%EC%9D%B8-%ED%8F%AC%ED%8A%B8-kill

**Comment by pronist** [comment, marked as answer]
@MJKim99 안녕하세요. 혹시, 레인보우 링크 텍스트를 해제해 보시겠어요?

**Reply by MJKim99** [reply]
@pronist  안녕하세요! 레인보우 링크 텍스트 해제 시 문제 없이 동작합니다. 감사합니다!!
혹시 추후 수정하실 계획이 있으실까욥?

**Reply by pronist** [reply]
@MJKim99 레인보우 링크 텍스트에 대한 대응 계획은 없습니다~

**Reply by MJKim99** [reply]
@pronist 넵! 확인 감사합니다~!
